lib/src/dartdoc_options.dart

```diff
@@ -1624,6 +1626,8 @@ Future<List<DartdocOption>> createDartdocOptions() async {
             'exist. Executables for different platforms are specified by '
             'giving the platform name as a key, and a list of strings as the '
             'command.'),
+    // TODO(jdkoren): unhide when this feature is in working order.
+    new DartdocOptionArgFile<String>('format', 'html', hide: true)
```

Comment:
Does this have to be both command line and options file? Just have a command line flag only?

Reply:
Done